doc/conf.py

```diff
@@ -71,10 +71,11 @@
     "gallery_dirs": ["gallery", "tutorials", "projections"],
     "subsection_order": ExplicitOrder(
         [
-            "../examples/gallery/line",
-            "../examples/gallery/coast",
-            "../examples/gallery/plot",
-            "../examples/gallery/grid",
+            "../examples/gallery/images",
+            "../examples/gallery/maps",
+            "../examples/gallery/lines",
+            "../examples/gallery/symbols",
+            "../examples/gallery/3d-plots",
```

Comment:
Any reason to put "images" in the first place? The following orders make more sense to me,

```suggestion
            "../examples/gallery/maps",
            "../examples/gallery/lines",
            "../examples/gallery/symbols",
            "../examples/gallery/images",
            "../examples/gallery/3d-plots",
```

Reply:
> Any reason to put "images" in the first place? The following orders make more sense to me,

Yes, this order is more sense to me, too.

Reply:
I was organizing it alphabetically; I'll make the change.